.vale/styles/OpenShiftAsciiDoc/SuggestAttribute.yml

```diff
@@ -0,0 +1,85 @@
+---
+extends: substitution
+level: suggestion
+message: "Use the AsciiDoc attribute '%s' rather than the plain text product term '%s', unless your use case is an exception."
+link: https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc
+scope: raw
+nonword: true
+action:
+  name: replace
+swap:
+  OpenShift Container Platform: "{product-title}"
+#  OpenShift: "{product-title}"
```

Comment:
```suggestion
OpenShift(?! Container Platform): "{product-title}"
```

Reply:
Sure, I can move my commented items off to a scratch space to tackle later :) Thanks for all your help. Should I merge commits in this repo?

Reply:
`OpenShift(?! Container Platform)` will catch "the OpenShift CLI (`oc`)", which we allow. 

However, I do think it would be great to replace that phrase with an attribute (like `{oc-first}` or something). That would probably knock out the vast majority of legitimate instances of "OpenShift" in the docs. 

I'm going to leave it alone for now but perhaps will propose replacing the phrase with a new attribute next week as a Shift Week project, which the aim of making another Vale rule.

Reply:
We can regex to handle both I think :)

Reply:
Yeah - though I have proposed [this new attribute](https://github.com/openshift/openshift-docs/pull/61060), so if that is approved we can approach from that angle :nerd_face: